cmd/draft/init.go

```diff
@@ -76,6 +80,7 @@ func newInitCmd(out io.Writer, in io.Reader) *cobra.Command {
 
 	f := cmd.Flags()
 	f.BoolVarP(&i.clientOnly, "client-only", "c", false, "install local configuration, but skip remote configuration")
+	f.BoolVarP(&i.ingress, "ingress", "i", false, "configure ingress")
```

Comment:
should we change this to --ingress-enabled so it's consistent with draftd's option flag?

Reply:
yeah let's do that.